rust/operator-binary/src/druid_controller.rs

```diff
@@ -738,6 +769,53 @@ fn add_hdfs_cm_volume_and_volume_mounts(
     }
 }
 
+fn get_ldap_secret_volume_and_volume_mounts_and_commands(
+    maybe_ldap_settings: &Option<DruidLdapSettings>,
+) -> (BTreeMap<String, (String, Volume)>, Vec<String>) {
+    let mut volumes = BTreeMap::new();
+    let mut commands = Vec::new();
+
+    if let Some(ldap_settings) = maybe_ldap_settings {
+        if let Some(credentials) = &ldap_settings.provider.bind_credentials {
+            let volume_name = credentials.secret_class.clone();
+            let secret_volume = VolumeBuilder::new(&volume_name)
+                .ephemeral(SecretOperatorVolumeSourceBuilder::new(volume_name.clone()).build())
+                .build();
+
+            volumes.insert(
+                volume_name.clone(),
+                (format!("/stackable/secrets/{volume_name}"), secret_volume),
+            );
+
+            let ldap_bind_user = format!("$(cat /stackable/secrets/{volume_name}/user)");
+            let ldap_bind_password = format!("$(cat /stackable/secrets/{volume_name}/password)");
+
+            const RUNTIME_PROPERTIES_PATH: &str = "/stackable/rwconfig/runtime.properties";
```

Comment:
I would suggest calling it `RUNTIME_PROPERTIES_FILE`
Please re-use
`pub const RW_CONFIG_DIRECTORY: &str = "/stackable/rwconfig";`
and
`pub const RUNTIME_PROPS: &str = "runtime.properties";`